src/FSharp.Core/list.fsi

```diff
@@ -243,7 +243,7 @@ module List =
     [<CompiledName("ChunkBySize")>]
     val chunkBySize: chunkSize:int -> list:'T list -> 'T list list
 
-    /// <summary>For each element of the list, applies the given function. Concatenates all the results and return the combined list.</summary>
+    /// <summary>For each element of the list, apply the given function. Concatenates all the results and return the combined list.</summary>
```

Comment:
Based on some other summaries I see in this PR, seems like they are all written in third person.
```suggestion
    /// <summary>For each element of the list, applies the given function. Concatenates all the results and returns the combined list.</summary>
```
Not sure about the comma. Maybe just "Applies the given function to each element of the list" sounds more natural, but I don't know English well enough to nitpick here.

Reply:
Looking back at it again, I think I agree with you. I'll leave this one out for now, although it still reads a bit odd to me.